src/main/java/com/codegym/cgshopeeapp/model/entity/User.java

```diff
@@ -0,0 +1,141 @@
+package com.codegym.cgshopeeapp.model.entity;
+
+import java.sql.Date;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+public class User {
+    private String emailUser;
+    private String nameUser;
```

Comment:
in class User, we understand properties of User, should rename to email, username, password, address,...